Svg/Clipping and Masking/SvgClipPath.cs

```diff
@@ -73,6 +77,32 @@ public Region GetClipRegion(SvgVisualElement owner)
             return new Region(GetClipRegionPath(owner));
         }
 
+        private RectangleF CalculateClipBounds()
```

Comment:
why is the reactangle implementing "GetBounds" differently?

Reply:
CliPath is not a VisualElement. I am calculating the bounds based on its children, Like the <svg> element does

Reply:
Sry... I was confused because of "Rectangle"... my bad